protoc-gen-twirp_php/templates/service/Client.php

```diff
@@ -47,6 +47,11 @@ final class {{ .Service | phpServiceName .File }}Client implements {{ .Service |
      */
     private $streamFactory;
 
+    /**
+     * @var boolean
+     */
+    private $doProtobufRequest = true;
```

Comment:
Feels like protobuf is the normal case and JSON is the special one, so I would probably switch this to something like:

```suggestion
    /**
     * @var bool
     */
    private $json = false;
```